python/lsst/obs/base/script/convertGen2RepoToGen3.py

```diff
@@ -172,7 +172,7 @@ def convert(gen2root, gen3root, instrumentClass, calibFilterType,
     instrument.applyConfigOverrides(ConvertRepoTask._DefaultName, convertRepoConfig)
     if convertRepoConfig.raws.instrument is None:
         convertRepoConfig.raws.instrument = instrumentClass
-    convertRepoConfig.raws.transfer = "symlink"
+    convertRepoConfig.raws.transfer = "auto"
```

Comment:
This really needs to be controllable from the command-line.  The right way to do that is to leverage the pex_config-override argument parsing that's in ctrl_mpexec, so maybe that can wait until we make this a `butler` subcommand.

But we also shouldn't let the lack of command-line flexibility now bias our choice of default behavior (especially down in daf_butler; see PR comment there).  It'd be better to add an argparse option here for transfer mode manually, even if we do later replace it with more general pex_config stuff.